x-pack/plugin/enterprise-search/src/test/java/org/elasticsearch/xpack/enterprisesearch/EnterpriseSearchPluginIT.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.enterprisesearch;
+
+import org.elasticsearch.action.admin.indices.create.CreateIndexResponse;
+import org.elasticsearch.action.get.GetResponse;
+import org.elasticsearch.action.index.IndexResponse;
+import org.elasticsearch.common.settings.MockSecureSettings;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.rest.RestStatus;
+import org.elasticsearch.test.ESIntegTestCase;
+import org.elasticsearch.xpack.enterprisesearch.action.decrypt.DecryptAction;
+import org.elasticsearch.xpack.enterprisesearch.action.decrypt.DecryptRequest;
+import org.elasticsearch.xpack.enterprisesearch.action.decrypt.DecryptResponse;
+import org.elasticsearch.xpack.enterprisesearch.action.encrypt.EncryptAction;
+import org.elasticsearch.xpack.enterprisesearch.action.encrypt.EncryptRequest;
+import org.elasticsearch.xpack.enterprisesearch.action.encrypt.EncryptResponse;
+import org.elasticsearch.xpack.enterprisesearch.crypto.CryptoService;
+import org.elasticsearch.xpack.enterprisesearch.setting.EntSearchField;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+
+
+@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
+public class EnterpriseSearchPluginIT extends ESIntegTestCase {
```

Comment:
Maybe `ESSingleNodeTestCase` would be a better choice, since you don't have to specify specific node settings. But, unsure of the fine details here 🤷